lib/keystore.hh

```diff
@@ -10,19 +10,21 @@ namespace rpm {
 
 class keystore {
 public:
+    const std::string name;
     virtual rpmRC load_keys(rpmtxn txn, rpmKeyring keyring) = 0;
     virtual rpmRC import_key(rpmtxn txn, rpmPubkey key, int replace = 1, rpmFlags flags = 0) = 0;
     virtual rpmRC delete_key(rpmtxn txn, rpmPubkey key) = 0;
 
     virtual ~keystore() = default;
+    keystore(std::string n="none"): name(n) {};
 };
 
 class keystore_fs : public keystore {
 public:
     virtual rpmRC load_keys(rpmtxn txn, rpmKeyring keyring);
     virtual rpmRC import_key(rpmtxn txn, rpmPubkey key, int replace = 1, rpmFlags flags = 0);
     virtual rpmRC delete_key(rpmtxn txn, rpmPubkey key);
-
+    keystore_fs(std::string n="fs"): keystore(n) {};
```

Comment:
Don't use a default argument, we don't want anybody calling eg the "fs" keystore with "rpmdb" as the name, ever. Just hardcode the name into the keystore constructor call.